src/services/VibeMap.js

```diff
@@ -527,9 +530,9 @@ module.exports = {
         const popular_bonus = 5
 
         // Weight distance & rating different than other fields
-        const vibe_factor = 2.0 
+        const vibe_factor = 1.0 
```

Comment:
is vibe_factor used again? What would it's purpose be, considering it's value is 1.0

Reply:
Not sure about the logic behind the reward for being openNow and openToday being the same (+ open_bonus). That would double the reward in all instances where item is openNow. On the flip side, this means something that is closed now, but was open earlier earlier today, would still receive half the reward of a place that is openNow. Reward should only be given if "open later today." Also, personally I think if a place is open now, it should be more than two times as valuable as open today. Either way, having two unique variables instead of just "const open_bonus" would give us more flexibility. We'd rather not recommend closed places especially if in app

Reply:
I would love to hear the ideation behind the choices for weights/arithmetic. The scaling overall seems a bit inconsistent to me considering there's a mix of addition and multiplication, large and small values, etc. I can just imagine a few scenarios where this might be problematic. We can talk more in detail another time though, no glaring major issues to me